src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -1703,27 +1703,25 @@ TEST_F(PolicyHandlerTest, DISABLED_OnSnapshotCreated_UrlAdded) {
 
 #ifdef PROPRIETARY_MODE
   ExtendedPolicyExpectations();
-#else
-  AppIdURL next_app_url = std::make_pair(0, 0);
+#else   // HTTP mode
```

Comment:
@ZhdanovP Why else is `HTTP` mode?

Reply:
@AByzhynar Because the EXTERNAL_PROPRIETARY_MODE is checked higher (line 1677)